lib/src/js/analyze.js

```diff
@@ -43,7 +43,7 @@ function getBehaviors(results, filePath) {
     if (behavior.contentHref != filePath) continue;
 
     behaviors.push({
-      name: behavior.is,
+      name: behavior.is.replace('Polymer.', ''),
```

Comment:
Ah, I see, without this you didn't have a normalized way to refer to the polymer names.  Makes sense to me now, thank you.